### PR TITLE
no

### PR DESCRIPTION
**Pull request by Iamgoofball**
no

### CODE REVIEW

**Comment by WJohn** [comment]
god damnit goof

**Comment by PKPenguin321** [comment]
Heh

**Comment by anconfuzedrock** [comment]
yea this is pretty funny

**Comment by Iamgoofball** [comment]
i got muted from OOC by mimicfaux for making funn of this

**Comment by MimicFaux** [comment]
You got muted for observing a round just to push dumb PR memes.

**Comment by Supermichael777** [comment]
P>.05 not statistically relevant

**Comment by Iamgoofball** [comment]
@MimicFaux 
And you refused to reply to any requests as to why I was muted, by rejecting my ahelps?
Excuse me? I was watching the round because of lighting, and was also wondering why the PR announcer wasn't displayinng any opened PRs.

**Comment by sawrge** [comment]
everyone in game caonically has autism already

**Comment by MimicFaux** [comment]
@Iamgoofball 
I was not rejecting your ahelps. As for why I didn't respond; you were already talking to Bobbington, and I was talking in asay everyone's opinions on the matter. This also doesn't seem like the proper place to talk about this.

**Comment by Iamgoofball** [comment]
You don't idle in supportbus, you refused to reply to me in game, and I'm banned from Policy Discussion. Too bad.

Then why didn't you outright tell me? You placed the mute, when the other person says they don't know why it was placed.

**Comment by Cyberboss** [comment]
Bad taste goof 👎

**Comment by KorPhaeron** [comment]
Healthy young greyshirt goes to medbay, gets pumped with massive shot full of spacecillan, doesn't feel good and changes - AUTISM. Many such cases!

**Comment by Iamgoofball** [comment]
@Cyberboss What's wrong with making fun of antivaxxer idiots who deny facts and science?

**Comment by Cyberboss** [comment]
Forgive me if I'm not really seeing the joke, if anything I'm seeing reinforcement.

But maybe I'm just feeling political

**Comment by factoryman942** [comment]
if it's a 100% chance then people will end up weaponising vaccines :p 
nerf pls

**Comment by Cyberboss** [comment]
It's a 1 percent chance

**Comment by optimumtact** [comment]
this doesn't cause autism, it causes brain damage.

**Comment by Iamgoofball** [comment]
# AUTISM

**Comment by Supermichael777** [comment]
you cant give players autism if they already have it

**Comment by Haevacht** [comment]
Get the fuck out goof, this isn't funny enough to be a meme pr. This is fucking stupid, and low effort even for you.

**Comment by ChangelingRain** [comment]
are you aware that i'm really tired of your shit yet
this is minimum-effort shitposting.

**Comment by Iamgoofball** [comment]
# grudge close

**Comment by Iamgoofball** [comment]
seriously it's fucking hillarious you get away with the abusive shit you pull

>get called out for abusing maintainer status to grudge remove shit you dont like
>close PRs of the person calling you out

**Comment by ChangelingRain** [comment]
you will not find anyone willing to reopen this get some better fucking memes you dullfaced hatless arsebutt.

**Comment by Haevacht** [comment]
Put effort into your meme prs and don't make them about topics that get people killed, goof.

**Comment by optimumtact** [comment]
[](http://puu.sh/u0Zaw/97cf10f781.png)

**Comment by MrPerson** [comment]
#rekt

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -91,6 +91,9 @@
 		for(var/datum/disease/D in M.viruses)
 			if(D.GetDiseaseID() in data)
 				D.cure()
+		if(prob(1) && istype(M, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = M
+			H.adjustBrainLoss(100)
```
Comment (MrStonedOne) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Shouldn't this be -100

Everybody knows autists are smarter then the causal neuraltypical.

**Comment by FantasticFwoosh** [comment]
TBH @Iamgoofball coding & creating a IC counter vaccine treatment (besides mannitol) with another attached commit would eliminate the point of the spessmen inhabiting the game.

> What's next? using buttered slices of toast strapped to your feet as anti-magboots for free-flight?

Real talk, this just helps traitor viro by spamming out useless-ass diseases since accidental cures for multiple diseases would be more damaging than the useless effects of sneezing with no symptoms.

**Comment by Niknakflak** [comment]
This is pushing a bad narrative.

**Comment by optimumtact** [comment]
This is a documented Alternative truth sorry niknak

**Comment by Arianya** [comment]
meme pr close

**Comment by FantasticFwoosh** [comment]
Jokes on you uncultured Americano, in Japan the 'ok' jesture means zero. 

Meaning the rate of zero approval for this PR as endorsed by yourself.

**Comment by optimumtact** [comment]
To make this more realistic we need to brainstorm an actual autism syndrome and some side-effects in game, thoughts?

**Comment by FantasticFwoosh** [comment]
Autism is too varied to pinpoint directly, i uh _talk from experience_

> Disallow hugging (social contact)
> Occasional minor tourette ticks
> Jitteryness (see above ticks)

If you really want to go there, it'll probably the most controversial thing to add to date.

**Comment by ChemicalRascal** [comment]
It might be interesting if it actually caused people to take damage by *being* hugged.

**Comment by Grazyn** [comment]
Tbh it's a good balance to vaccines, after all when was the last time you saw a true deadly epidemic on the station after viro got nerfed? This at least gives a downside to mass producing vaccines, you need to make mannitol too and make mannitol+vaccine pills instead of just pumping out a ton of 1u vaccine pills

**Comment by Arianya** [comment]
Or, hot opinion here...

You give vaccines mild toxins damage. Bam, same thing without making a shitty joke about autism.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (Shadowlight213) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Probably should give tk too.

Comment (RemieRichards) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: and turn them into an ayy, since they can't talk to people anyway.

**Comment by uraniummeltdown** [comment]
tfw to intelligent for vaccines

**Comment by Iamgoofball** [comment]
Antivaxxers are retarded though and poking fun at them is great

**Comment by Kraseo** [comment]
i blow up and this gay meme dies

**Comment by KorPhaeron** [comment]
we have to respect the rp laws I don't have a counter for krasos attack

**Comment by Iamgoofball** [comment]
Fugg

**Pull request closed without merging**